docs/playbook-reference/defining-playbooks/external-playbook-repositories.rst

```diff
@@ -109,6 +116,27 @@ The ``key`` parameter must contain a ``base64`` encoded deployment key with ``re
         ewfrcfsfvC1rZXktdjEAAAAABG5vb.....
         -----END OPENSSH PRIVATE KEY-----
 
+Loading Actions from an external Python Package
+---------------------------------------------------
+
+For external Python packages, just specify an URL starting with http(s), and ending with
+either .tar.gz or .tgz.
+
+.. code-block:: yaml
+
+    playbookRepos:
+      web_playbooks:
+        url: "https://my-domain.com/bla/web-playbooks.tgz"
+        http_headers: # optional, may be used for auth
+          Authorization: Bearer XXXYYY
+        # pip_install: true      # optional: load this playbook's dependencies (default True)
```

Comment:
I think this might be a little confusing for users.
The `pip_install` and `build_isolation` are related to all forms of playbooks loading, and not only the `tgz`

So I think it should be removed from here, and documented on a separate section, explaining that this should be used when installing playbooks on air gapped envs, or similar.
WDYT?

Reply:
created a different section for it